{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"!="},{"kind":"text","text":" "},{"kind":"text","text":"("},{"kind":"internalParam","text":"lhs"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Self"},{"kind":"text","text":", "},{"kind":"internalParam","text":"rhs"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Self"},{"kind":"text","text":") -> "},{"kind":"typeIdentifier","text":"Bool","preciseIdentifier":"s:Sb"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/awsaccount\/internalserverexception\/!=(_:_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AWSAccount\/documentation\/AWSAccount\/InternalServerException\/!=(_:_:)","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"Equatable.!=(_:_:)"},{"type":"text","text":"."}],"kind":"symbol","metadata":{"role":"symbol","title":"!=(_:_:)","roleHeading":"Operator","fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"!="},{"kind":"text","text":" "},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"Self"},{"kind":"text","text":", "},{"kind":"typeIdentifier","text":"Self"},{"kind":"text","text":") -> "},{"kind":"typeIdentifier","text":"Bool","preciseIdentifier":"s:Sb"}],"symbolKind":"op","externalID":"s:SQsE2neoiySbx_xtFZ::SYNTHESIZED::s:10AWSAccount23InternalServerExceptionV","extendedModule":"Swift","modules":[{"name":"AWSAccount"}]},"hierarchy":{"paths":[["doc:\/\/AWSAccount\/documentation\/AWSAccount","doc:\/\/AWSAccount\/documentation\/AWSAccount\/InternalServerException","doc:\/\/AWSAccount\/documentation\/AWSAccount\/InternalServerException\/Equatable-Implementations"]]},"references":{"doc://AWSAccount/documentation/AWSAccount":{"role":"collection","title":"AWSAccount","abstract":[],"identifier":"doc:\/\/AWSAccount\/documentation\/AWSAccount","kind":"symbol","type":"topic","url":"\/documentation\/awsaccount"},"doc://AWSAccount/documentation/AWSAccount/InternalServerException/!=(_:_:)":{"role":"symbol","title":"!=(_:_:)","fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"!="},{"kind":"text","text":" "},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"Self"},{"kind":"text","text":", "},{"kind":"typeIdentifier","text":"Self"},{"kind":"text","text":") -> "},{"kind":"typeIdentifier","text":"Bool","preciseIdentifier":"s:Sb"}],"abstract":[],"identifier":"doc:\/\/AWSAccount\/documentation\/AWSAccount\/InternalServerException\/!=(_:_:)","kind":"symbol","type":"topic","url":"\/documentation\/awsaccount\/internalserverexception\/!=(_:_:)"},"doc://AWSAccount/documentation/AWSAccount/InternalServerException/Equatable-Implementations":{"role":"collectionGroup","title":"Equatable Implementations","abstract":[],"identifier":"doc:\/\/AWSAccount\/documentation\/AWSAccount\/InternalServerException\/Equatable-Implementations","kind":"article","type":"topic","url":"\/documentation\/awsaccount\/internalserverexception\/equatable-implementations"},"doc://AWSAccount/documentation/AWSAccount/InternalServerException":{"role":"symbol","title":"InternalServerException","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"InternalServerException"}],"abstract":[{"type":"text","text":"The operation failed because of an error internal to Amazon Web Services. Try your operation again later."}],"identifier":"doc:\/\/AWSAccount\/documentation\/AWSAccount\/InternalServerException","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"InternalServerException"}],"url":"\/documentation\/awsaccount\/internalserverexception"}}}